buckaroo3.php

```diff
@@ -903,7 +926,12 @@ public function hookPaymentOptions($params)
                 'country'                 => Country::getIsoById(Tools::getCountry()),
                 'afterpay_show_coc'       => $this->showAfterpayCoc($cart),
                 'billink_show_coc'        => $this->showBillinkCoc($cart),
-                'idealIssuers'           => (new IssuersIdeal())->get()
+                'idealIssuers'            => (new IssuersIdeal())->get(),
+                'idealShowIcons'          => Config::get('BUCKAROO_IDEAL_DISPLAY_TYPE'),
+                'paybybankIssuers'        => (new PayByBank())->getIssuerList(),
```

Comment:
better if we move the issuer retrieving in a different class, or swich the retrieving method to a static method, same with credicards bellow